lib/validation.js

```diff
@@ -9,10 +9,25 @@ const bodySchema = Symbol('body-schema')
 const querystringSchema = Symbol('querystring-schema')
 const outputSchema = Symbol('output-schema')
 const paramsSchema = Symbol('params-schema')
+const responseSchema = Symbol('response-schema')
 
 const schemas = require('./schemas.json')
 const inputSchemaError = fastJsonStringify(schemas.inputSchemaError)
 
+function getValidatorForStatusCodeSchema (statusCodeDefinition) {
+  return fastJsonStringify(statusCodeDefinition)
+}
+
+function getResponseSchema (responseSchemaDefinition) {
+  var schemas = { }
+  var statusCodes = Object.keys(responseSchemaDefinition)
+  for (var i = 0; i < statusCodes.length; i++) {
+    var statusCode = statusCodes[i]
+    schemas[statusCode] = getValidatorForStatusCodeSchema(responseSchemaDefinition[statusCode])
+  }
```

Comment:
This should be on a non-hot path, i.e. it is not in the rendering pipeline. If that's the case you can use a `forEach`, it's much more readable.